### DISCUSSION TITLE
add support g255f

### DISCUSSION THREAD

**Discussion by neoochii** [xanderfrangos/twinkle-tray#852] [Monitor Compatibility Issues]
add support g255f

**Comment by xanderfrangos** [comment]
Hi @neoochii,

There's no way for me to add support for individual monitor models. They're either compatible or they're not, depending on the model and your PC configuration. I don't own a G255F, so I can't even tell you why it's not working for you.

I'm going to move this to Discussions, in case another G255F owner can confirm if it works or not.
